js/src/main/scala/Main.scala

```diff
@@ -42,7 +42,6 @@ object Main {
           "Parse error: " + extra.trace().msg +
             s" at line $lineNum:<BLOCKQUOTE>$lineStr</BLOCKQUOTE>"
         case Success(pgrm, index) =>
-          println(s"Parsed: $pgrm")
```

Comment:
Is this removal on purpose? I find it quite useful to debug what's going on, when a problem occurs.

Reply:
It’s accidental. I was confused by a lot of outputs in console.